docker/compose.yml

```diff
@@ -1,21 +1,30 @@
-version: '3'
+version: "3"
 services:
   opensearch:
     image: rps-os
     container_name: opensearch
     environment:
-      - discovery.type=single-node
+      - OPENAI_API_KEY
+      # To help with debugging issues with opensearch starting.
+      - DEBUG
+      - NOEXIT
     ports:
       - 9200:9200
       - 9600:9600
     networks:
       - os-net
+    volumes:
+      - rpsos:/usr/share/opensearch/data
```

Comment:
I'm not sure why adding SSL needs a volume, but whatever.

Reply:
that was just so I didn't have to keep ingesting an index and downloading models between test runs